src/extractDates.js

```diff
@@ -6,11 +6,18 @@ export default extractDates
 const parser = new chrono.Chrono
 parser.refiners.push(wholeMonth)
 
-function extractDates(str) {
+function extractDates(str, timezoneOffset) {
```

Comment:
I initially had a default. It would have needed to be the current timezone offset (moment().utcOffset()) otherwise it would always parse as UTC when no offset was provided. I thought it would be better to not have to provide one at all which means we don't have to change the other code relying on the parser. 

Then I changed it to the condition block as it was consistent with the `tickbin-parser` and I thought it was cleaner looking. V.S.

```javascript
function extractDates(str, timezoneOffset = moment().utcOffset()) {
  ...
}
```